src/traces/image/hover.js

```diff
@@ -0,0 +1,79 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Fx = require('../../components/fx');
+var Lib = require('../../lib');
+var constants = require('./constants');
+
+// var Axes = require('../../plots/cartesian/axes');
```

Comment:
:zap:

Reply:
Done in 73c0740f53fc82a70196d1993ba2824db2be85a3